pkg/apb/watch_pod.go

```diff
@@ -60,3 +68,23 @@ func watchPod(podName string, namespace string) error {
 
 	return fmt.Errorf("Timed out while watching pod %s for completion", podName)
 }
+
+func errorPullingImage(conds []apiv1.ContainerStatus) bool {
+	if len(conds) < 1 {
+		log.Warningf("unable to get container status for APB pod")
+		return false
+	}
+	// We should expect only a single container for our APB pod.
+	// If this assumption changes then we may need to update this code.
+	// Basis for the image strings is here:
+	// https://github.com/kubernetes/kubernetes/blob/886e04f1fffbb04faf8a9f9ee141143b2684ae68/pkg/kubelet/images/types.go#L27
+	status := conds[0].State.Waiting
+
+	if status.Reason == "ErrImagePull" {
```

Comment:
Might as well use ```images.ErrImagePull.Error()``` and ```images.ErrImagePullBackOff.Error()``` since it's where the string comes from.

Reply:
I would prefer not to use something that is not in the kubernetes client-go or kubernetes api. 

The images errors that are referenced in the core kuberentes package, I think that eventually, we could remove this dependency.

Reply:
wfm